### DISCUSSION TITLE
excel doesn't pop up

### DISCUSSION THREAD

**Discussion by adkprakash** [PaddlePaddle/PaddleOCR#13535] [Q&A]
after table recognition get triggered it shows this error, even though pywin32 is installed:
file name in openNext is  C:\Users\Acer\Pictures\data\image.jpg
DirPath in importDirImages is C:/Users/Acer/Pictures/data
Can not save excel file, maybe Permission denied (.xlsx is being occupied)
CANNOT OPEN .xlsx. It could be the following reasons: .xlsx is not existed
time cost:  4.430507659912109

**Comment by UserWangZz** [comment]
This looks like the program cannot save the excel file, try changing the output path and see if there is a program occupying it.
